### PR TITLE
Added Per-Grade Outfit Support

### PR DESCRIPTION
**Pull request by nick-perry14**
Added a simple additional grade check to allow different grades of the same jobs to be given different preset outfits.  This allows for rank pins, different clothing options, etc. for higher grade players

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -540,283 +539,289 @@ Config.ClothingRooms = {
...
-                    ["pants"]       = { item = 34,  texture = 0 },  -- Pants
-                    ["shoes"]       = { item = 29,  texture = 0 },  -- Shoes
-                    ["mask"]        = { item = 0,   texture = 0 },  -- Mask
-                    ["hat"]         = { item = -1,  texture = 0 },  -- Hat
-                    ["glass"]       = { item = 0,   texture = 0 },  -- Glasses
-                    ["ear"]         = { item = 0,   texture = 0 },  -- Ear accessories
-                },
-            },
+        ["female"] = { -- Gender
+			[0] -- Grade Level
```
Comment (mk3ext) [inline config.lua:769]: This should be `[0] = {` rather than `[0]` as it will cause an error

File: client/main.lua [hunk 2]
```diff
@@ -458,7 +458,7 @@ Citizen.CreateThread(function()
                                         if QBCore.Functions.GetPlayerData().charinfo.gender == 1 then gender = "female" end
                                         QBCore.Functions.TriggerCallback('qb-clothing:server:getOutfits', function(result)
                                             openMenu({
-                                                {menu = "roomOutfits", label = "Presets", selected = true, outfits = Config.Outfits[PlayerData.gang.name][gender]},
+                                                {menu = "roomOutfits", label = "Presets", selected = true, outfits = Config.Outfits[PlayerData.gang.name][gender][PlayerData.job.grade.level]},
```
Comment (mk3ext) [inline client/main.lua:461]: This shouldn't use `PlayerData.job.grade.level` since its a gang it should be `PlayerData.gang.grade.level`

**Review by mk3ext — CHANGES_REQUESTED** [review]
Added comments to possibly some issues I found

File: client/main.lua (hunk 2 above)
Comment (nick-perry14) [reply client/main.lua:461]: Thanks.  Will Fix In Second Update.  Standby

**Comment by nick-perry14** [comment]
Comments by @mk3ext resolved

**Comment by MonkeyWhisper** [comment]
Tested and working.

**Pull request merged**